app/models/user.rb

```diff
@@ -433,11 +433,12 @@ def self.create_with_omniauth(auth)
     email_prefix = auth["info"]["email"].tr('.', '_').split('@')[0]
     email_prefix = auth["info"]["email"].tr('.', '_').split('@')[0] + Array.new(2) { charset.sample }.join until User.where(username: email_prefix).empty?
     puts(auth)
```

Comment:
Shall we remove this log line? 

Also, is `Array.new(12)` sufficient? I was able to use `SecureRandom.random_bytes(32)` as suggested by @ryzokuken -- did that not work?